Framework/API/src/Algorithm.cpp

```diff
@@ -460,6 +461,7 @@ bool Algorithm::execute() {
     logAlgorithmInfo();
 
   // Check all properties for validity
+  float timingInit = timer.elapsed();
```

Comment:
I find the API for `Timer::elapsed` to be counter intuitive. Reading the code I initially thought that each time point you assign was a total offset since `time` is constructed, however the reset argument defaults to true. Thus your code gives the correct behaviour. I think an improvement to readability for future maintainers would be to explicitly pass the flag indicating that the timer is reset as part of the `elapsed` call.

Reply:
:+1: